grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/extension/TestExtension.kt

```diff
@@ -3,14 +3,18 @@ package com.grab.grazel.extension
 import groovy.lang.Closure
 
 
-private const val DEFAULT_TEST_SIZE = "medium"
 private const val DEFAULT_ROBOLECTRIC_VERSION = "4.4"
+private const val DEFAULT_ROBOLECTRIC_STRIP_PREFIX = "robolectric-bazel-4.4"
+private const val DEFAULT_ROBOLECTRIC_URL = "https://github.com/roscrazy/robolectric-bazel/releases/tag/4.4.2.tar.gz"
 
 data class RobolectricExtension(
+    var stripPrefix: String = DEFAULT_ROBOLECTRIC_STRIP_PREFIX,
+    var archiveUrl: String = DEFAULT_ROBOLECTRIC_URL,
     var version: String = DEFAULT_ROBOLECTRIC_VERSION
 )
 
-data class AndroidTestExtension (
+
+data class AndroidTestExtension(
     var robolectric: RobolectricExtension = RobolectricExtension()
```

Comment:
This can be `val` since we will only allow properties of `RobolectricExtension` to be modified and not the extension itself.